fe/src/main/java/org/apache/doris/analysis/SlotRef.java

```diff
@@ -162,6 +162,12 @@ public String toSqlImpl() {
             return tblName.toSql() + "." + label + sb.toString();
         } else if (label != null) {
             return label + sb.toString();
+        } else if (desc.getSourceExprs() != null) {
+            for (Expr expr : desc.getSourceExprs()) {
+                sb.append(expr.toSql());
+                sb.append(" ");
+            }
+            return sb.toString();
```

Comment:
How about return both `slot id` and `expr`. I'm afraid that `slot id` may be used for some tracing debug.

And please fix all related unit tests after you change here.

Reply:
How about slot ID and expr? I need MD5 of SQL as the key of cache instead of slot ID. I'm trying to fix the other unit tests